testgrid/tgapi/pkg/handlers/master_join.go

```diff
@@ -0,0 +1,34 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/replicatedhq/kurl/testgrid/tgapi/pkg/logger"
+	"github.com/replicatedhq/kurl/testgrid/tgapi/pkg/testinstance"
+)
+
+type JoinMasterRequest struct {
```

Comment:
i understand you are only following a pattern here but rather than perpetuate the problem...

can we use inclusive language?

primary > master
secondary > worker